### PR TITLE
[12] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by elizabethbusola**
## Task: Styled the contact us Page using CSS
-  I styled the formerly created contact us page form using CSS
webpage_contact
tablet_contact
tablet_contact2
mobile-contact
mobile-contact2

### CODE REVIEW

File: submissions/busolaAdedoyin/css/style2.css [hunk 1]
```diff
@@ -0,0 +1,229 @@
+ body {
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/css/style2.css:1]: well done. Rename this file from `style2.css` to `contact.css` instead

**Comment by ToluAkin** [comment]
@elizabethbusola Well done! I think you should add some padding between the input line and the text beneath the input field.

File: submissions/busolaAdedoyin/css/style2.css (hunk 1 above)
Comment (elizabethbusola) [reply submissions/busolaAdedoyin/css/style2.css:1]: @adaugochi I have renamed the file to `contact.css`

**Comment by elizabethbusola** [comment]
> @elizabethbusola Well done! I think you should add some padding between the input line and the text beneath the input field.

@ToluAkin I have done it

**Pull request merged**